src/sortStudents.ts

```diff
@@ -1,16 +1,80 @@
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+export function sortStudents(
+  students: Student[],
+  sortBy: SortType,
+  order: SortOrder,
+): Student[] {
+  function getAverageGrade(student: Student): number {
+    return student.grades
+      .reduce((sum: number, el: number) => (
```

Comment:
`el` is not so meaningful name. `grade` - a good choice, `current` - also could be.
```suggestion
      .reduce((sum: number, grade: number) => (
```